### PR TITLE
Support input files that contain multiple VM host/network policies

### PR DESCRIPTION
**Pull request by wazir-ahmed**
Signed-off-by: Wazir Ahmed <wazir@accuknox.com>

### CODE REVIEW

File: vm/policy.go [hunk 1]
```diff
@@ -107,59 +110,71 @@ func PolicyHandling(t string, path string, o PolicyOptions, httpAddress string,
...
-		}
+		var hostPolicy tp.K8sKubeArmorHostPolicy
+		var networkPolicy v2.CiliumNetworkPolicy
+		var policyEvent interface{}
 
-	}
+		if k.Kind == KubeArmorHostPolicy {
+			err = json.Unmarshal(js, &hostPolicy)
+			if err != nil {
+				return err
```
Comment (nyrahul) [inline vm/policy.go:137]: error handling in this function is tricky!
What happens if one of the policy unmarshal fails and other were ok. All the policies before the failed policies were applied.

Comment (seswarrajan) [reply vm/policy.go:137]: Instead we can log the error and continue.

**Review by seswarrajan — CHANGES_REQUESTED** [review]

File: vm/policy.go (hunk 1 above)
Comment (wazir-ahmed) [reply vm/policy.go:137]: > All the policies before the failed policies were applied.

 @nyrahul Yes. This is similar to how `kubectl` works. If we have 5 policies, out of which the first three policies are good and the fourth one is invalid, then `kubectl` will apply the first three policies successfully and throw error while processing the fourth one.

Comment (wazir-ahmed) [reply vm/policy.go:137]: @seswarrajan The error returned from this function will be printed in the user console.

**Review by nyrahul — APPROVED** [review]
LGTM

**Pull request merged**